sklearn/multiclass.py

```diff
@@ -525,6 +525,13 @@ class OneVsOneClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
     estimators_ : list of ``n_classes * (n_classes - 1) / 2`` estimators
         Estimators used for predictions.
 
+    coef_ : ndarray of shape (1, n_features) or (n_classes, n_features)
+        Coefficient of the features in the decision function.
+
+    intercept_ : float or ndarray of shape (n_targets,)
```

Comment:
This 
```suggestion
    intercept_ : ndarray of shape (1, 1) or (n_classes, 1)
```

If `y` is binary, the shape is `(1, 1)` otherwise `(n_classes, 1)`